server/src/utils/getConfig.ts

```diff
@@ -14,12 +14,54 @@ import {
   VV_WAITING_TITLE_ENV_NAME,
   VV_LOGO_URL_ENV_NAME,
   VV_POSTCALL_SURVEY_TYPE_ENV_NAME,
-  VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME
+  VV_POSTCALL_SURVEY_OPTIONS_SURVEYURL_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_PROMPT_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TYPE_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_TITLE_ENV_NAME,
+  VV_POSTCALL_SURVEY_ONEQUESTIONPOLL_SAVE_BUTTON_TEXT_ENV_NAME
 } from '../constants';
 
-import { ServerConfigModel, ClientConfigModel, PostCallSurveyType, PostCallConfig } from '../models/configModel';
+import {
+  ServerConfigModel,
+  ClientConfigModel,
+  PostCallSurveyType,
+  PostCallConfig,
+  MSFormsSurveyOptions,
+  OneQuestionPollOptions,
+  OneQuestionPollType,
+  CustomSurveyOptions
+} from '../models/configModel';
 import { getDefaultConfig } from './getDefaultConfig';
 
+const getMSFormsOptions = (defaultConfig: any): MSFormsSurveyOptions => {
```

Comment:
Can we hard type this to `ServerConfigModel` instead of `any` ?

Reply:
Done.